lib/utils/util.js

```diff
@@ -0,0 +1,59 @@
+/** @flow */
+
+import {spawn} from 'child_process';
+import {window, workspace} from 'vscode'
+
+const NODE_NOT_FOUND = '[Flow] Cannot find node in PATH. The simpliest way to resolve it is install node globally'
+const FLOW_NOT_FOUND = '[Flow] Cannot find flow in PATH. Try to install it by npm install flow-bin -g'
+
+function getPathToFlow(): string {
+	return workspace.getConfiguration('flow').get('pathToFlow')
+}
+
+export function isFlowEnabled() {
+	return workspace.getConfiguration('flow').get('enabled')
+}
+
+export function checkNode() {
```

Comment:
@nmote If I use nvm to install flow-bin and don't have global node, flow-bin will not work at all. I think that we can try to execute flow to check is it require node or not, but it's not good for performance reasons (because we will try to start flow before we declare any Providers).

But If you want I can make settings for disable this check